src/app/routes/topic/getInitialData/index.js

```diff
@@ -1,11 +1,28 @@
+import { BFF_FETCH_ERROR } from '#lib/logger.const';
+import { INTERNAL_SERVER_ERROR } from '#lib/statusCodes.const';
+import nodeLogger from '#lib/logger.node';
 import { fixturePromos } from '#pages/TopicPage/fixtures';
+import fetchPageData from '../../utils/fetchPageData';
 
-export default () => {
-  return {
-    status: 200,
-    pageData: {
-      title: 'hello world',
-      promos: fixturePromos(),
-    },
-  };
+const logger = nodeLogger(__filename);
+
+export default async ({ getAgent, service }) => {
+  const agent = await getAgent();
+  try {
+    const path = process.env.BFF_PATH;
+    const { status, json } = await fetchPageData({ path, agent });
```

Comment:
I think you'll need to add the service and variant parameters to the path here?

Reply:
I'm trying not to do everything in one PR. I've included the param work in this ticket - https://github.com/bbc/simorgh-infrastructure/issues/1543.

Reply:
no probs, make sense that looks a great ticket 👍